playground/Qdrant/Qdrant.ServiceDefaults/Extensions.cs

```diff
@@ -0,0 +1,126 @@
+using Microsoft.AspNetCore.Builder;
```

Comment:
The playground samples share a service defaults.

Reply:
Fixed in next update